### ISSUE TITLE
Tribal Nation drop down HTML issue

### ISSUE THREAD

**Issue by lizzycederberg** [usnationalarchives/nara-bia-photos#256]
The select featured tribal nation dropdown doesn’t follow the intuitive tabbing order because, though it appears “first” in the content, it is second in the html. As with the other dropdown, it is missing the focus indicator.
